Sources/PointFree/Routers/PointFreeRoutes.swift

```diff
@@ -61,15 +81,13 @@ private let routers: [Router<Route>] = [
     <¢> get %> queryParam("success", opt(.bool)) <% end,
 
   Route.iso.invite <<< Route.Invite.iso.accept
-    // TODO: double raw representable is needed cause we have to go String -> UUID -> Tagged. Might need
-    //       a tagged combinator?
-    <¢> post %> lit("invites") %> pathParam((._rawRepresentable) >>> (._rawRepresentable)) <% lit("accept") <% end,
+    <¢> post %> lit("invites") %> pathParam((.uuid) >>> (.tagged)) <% lit("accept") <% end,
```

Comment:
and added a `uuid` parser so that we dont have to do `(._rawRepresentable) >>> (._rawRepresentable)`